charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             - name: "ACCEPT_EULA"
               value: {{ .Values.agent.acceptEula | quote }}
             - name: "TargetName"
-              value: {{ .Values.agent.targetName | quote }}
+              value: {{ .Values.agent.name | quote }}
             - name: "ServerCommsAddresses"
               value: {{ join "," .Values.agent.serverCommsAddresses | quote }}
             - name: "ServerCommsAddress"
```

Comment:
This can be removed as part of the major bump.

@scme0 please let me know if there are other fields that can be cleaned up.

Reply:
The two that come to mind are:
- `ServerCommsAddress` can be removed (we'll just use `ServerCommsAddresses` all the time instead)
- `targetRoles` should use the word `tags` instead of roles.

Reply:
I'll create a server PR to change the UI to always use `ServerCommsAddresses`, even if you only have a single value.

Reply:
@APErebus Are there any other properties that we should change in v2?